lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -1257,6 +1293,15 @@ states:
       fail-with-no-ci:
         targetState: MITIGATION_PRE_EXPERIAN_KBV_TRANSITION_PAGE
 
+  MITIGATION_PP_CRI_DWP_KBV:
+    response:
+      type: cri
+      criId: dwpKbv
+    parent: CRI_STATE
+    events:
+      next:
+        targetState: MITIGATION_CRI_DWP_KBV
```

Comment:
I don't think we need this intermediary state? can just go straight to `MITIGATION_CRI_DWP_KBV`
HMRC KBV only needs it because it has to go through NINO first